.utility/generate_rst_from_md.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""Script to generate a rst file automatically from an markdown file.
+
+Usage
+-----
+generate_rst_from_md.py [-h] markdown rst
+
+positional arguments:
+  markdown    Path of the source markdown file
+  rst         Path where generated rst should be stored
+
+optional arguments:
+  -h, --help  show this help message and exit
+"""
+import argparse
+import pypandoc
+
+def download_pandoc():
+    """Download pandoc if not already installed"""
+    try:
+        # Check whether it is already installed
+        pypandoc.get_pandoc_version()
+    except OSError:
+        # Pandoc not installed. Let's download it.
+        pypandoc.download_pandoc()
+
+        # Hack to delete the downloaded file from the folder,
+        # otherwise it could get accidently committed to the repo
+        # by other scripts in the repo.
+        import sys
+        import os
+        platform = sys.platform
+        if platform.startswith('linux'):
+            platform = 'linux'
+        url = pypandoc.pandoc_download._get_pandoc_urls()[0][platform]
+        filename = url.split('/')[-1]
+        os.remove(filename)
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('markdown', help='Path of the source markdown file')
+    parser.add_argument('rst', help='Path where generated rst should be stored')
+    args = parser.parse_args()
+    pypandoc.download_pandoc()
```

Comment:
It would be better to add line number 44 and 45 as a separate function so that it could be used in other places as well.